### PR TITLE
Fixes #88: Removes override of `_createRipple`.

### LINKED ISSUE #88: paper-radio-button should reference PaperInkyFocusBehavior instead of PaperInkyFocusBehaviorImpl
See https://github.com/PolymerElements/paper-radio-button/blob/05e74bb/paper-radio-button.html#L220.

PaperInkyFocusBehaviorImpl should only be referenced internally by PaperInkyFocusBehavior.

### PR DESCRIPTION
**Pull request by bicknellr**

### CODE REVIEW

File: paper-radio-button.html [hunk 1]
```diff
@@ -209,10 +209,8 @@
         }
       },
 
-      // create the element ripple inside the `radioContainer`
-      _createRipple: function() {
+      ready: function() {
```
Comment (notwaldorf) [inline paper-radio-button.html:212]: maybe `attached` instead?

**Comment by notwaldorf** [comment]
LGTM w one suggestion

File: paper-radio-button.html (hunk 1 above)
Comment (bicknellr) [inline paper-radio-button.html:212]: `this.$` is available at ready though, right?

Comment (notwaldorf) [inline paper-radio-button.html:212]: Sure, but if you're not in the document it doesn't really matter, does it?

Comment (bicknellr) [inline paper-radio-button.html:212]: Yeah, but then it runs on every attachment. The cases where create is run without attached immediately after (it's not hard-coded into a template or something) seem like they'd be more likely to attach / detach multiple times.

Comment (bicknellr) [inline paper-radio-button.html:212]: Arguably, this was right before... maybe I should drop this PR.

Comment (notwaldorf) [inline paper-radio-button.html:212]: Oh you're right about re-attachment. 👍

**Comment by notwaldorf** [comment]
LGTM

**Pull request merged**